packages/aragon-wrapper/src/rpc/handlers/cache.js

```diff
@@ -1,6 +1,8 @@
 export default function (request, proxy, wrapper) {
   const cacheKey = `${proxy.address}.${request.params[1]}`
   if (request.params[0] === 'get') {
+    // FIXME Isn't this obsolete?
+    // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/observe
     return wrapper.cache.observe(cacheKey)
```

Comment:
I think we can remove this comment; we're using `cache.observe()`, not the "standard" `observe`.